guides/micronaut-azure-cloud/micronaut-azure-cloud.adoc

```diff
@@ -21,7 +21,7 @@ Create an application using the https://docs.micronaut.io/latest/guide/#cli[Micr
 
 [source,bash]
 ----
-mn create-app example.micronaut.micronautguide --build=@build@ --lang=@lang@
+mn @cli-command@ example.micronaut.micronautguide --build=@build@ --lang=@lang@
 ----
 
 NOTE: If you don't specify the `--build` argument, Gradle is used as the build tool. +++<br/>+++ If you don't specify the `--lang` argument, Java is used as the language. For this guide, Maven is the required build tool.
```

Comment:
Should it be made more clear that the user needs to specify Maven as the build tool? Something like ...

```suggestion
NOTE: If you don't specify the `--build` argument, Gradle is used as the build tool. +++<br/>+++ If you don't specify the `--lang` argument, Java is used as the language. Because Maven is the required build tool for this guide, it must be specified.
```